src/files.jl

```diff
@@ -86,6 +88,7 @@ function Base.read!(file::TiffFile, arr::AbstractArray)
     end
 end
 
+# This is type-piracy
```

Comment:
I noticed this first by reading the code, but as I changed method dispatch I sometimes hit it in real life. I think it's pretty important to eliminate this before it becomes a FileIO default.

Reply:
Ack. Good catch. I need to just delete this method and figure out a different way to detect strided bilevel TIFFs

Reply:
Also, I'm 99% sure replacing io with `tf:TiffFile` would still work and fix the type piracy.